config/quality.gradle

```diff
@@ -15,6 +15,8 @@ task checkstyle(type: Checkstyle) {
     source 'src'
     include '**/*.java'
     exclude '**/gen/**'
+    exclude '**/androidTest/**'
+    exclude '**/test/**'
```

Comment:
The cucumber tests have very, very long names and the unit tests can have a lot of methods. We can also suppress specific warnings in the tests if you like.